sql/execution_pipeline_impl.cc

```diff
@@ -743,6 +845,29 @@ take()
             } while (r && r->values.back() < lField);
         }
     }
+
+    //Return unmatched rows if we have a LEFT/RIGHT/OUTER join
+    //Fill unmatched with empty values
+    if (outerLeft && l)
+    {
+        l->values.pop_back();
+        l->values.emplace_back(ExpressionValue("", Date()));
```

Comment:
Date() is certainly wrong.  It should be Date::notADate().  Date() returns 1-1-1970.